nipype/interfaces/fsl/tests/test_model.py

```diff
@@ -6,23 +6,23 @@
 import pytest
 import nipype.interfaces.fsl.model as fsl
 from nipype.interfaces.fsl import no_fsl
+from pathlib import Path
 
 
 @pytest.mark.skipif(no_fsl(), reason="fsl is not installed")
 def test_MultipleRegressDesign(tmpdir):
-    tmpdir.chdir()
     foo = fsl.MultipleRegressDesign()
     foo.inputs.regressors = dict(
         voice_stenght=[1, 1, 1], age=[0.2, 0.4, 0.5], BMI=[1, -1, 2]
     )
     con1 = ["voice_and_age", "T", ["age", "voice_stenght"], [0.5, 0.5]]
     con2 = ["just_BMI", "T", ["BMI"], [1]]
-    foo.inputs.contrasts = [con1, con2, ["con3", "F", [con1, con2]]]
+    foo.inputs.contrasts = [con1, con2, ["con3", "F", [con1, con2]], ["con4", "F", [con2]]]
     res = foo.run()
 
     for ii in ["mat", "con", "fts", "grp"]:
         assert (
-            getattr(res.outputs, "design_" + ii) == tmpdir.join("design." + ii).strpath
+            os.path.exists(eval('res.outputs.design_'+ii))
         )
 
     design_mat_expected_content = """/NumWaves       3
```

Comment:
To avoid further `eval`s or using `locals()` (also fine, but why?), we can make these dict entries:

```suggestion
    expected_content = {}

    expected_content["design_mat"] = """/NumWaves       3
```

And so on for the other `*_expected_content` variables.